proposals/0000-dollar-type-operator-as-default.rst

```diff
@@ -0,0 +1,45 @@
+.. proposal-number::
+
+.. trac-ticket::
+
+.. implemented::
+
+Dollar type operator as default
+===============================
+
+Define the dollar type operator (`type a $ b = a b`) as a default when using
+the `TypeOperators` extension.
+
+Motivation
+----------
+
+Though, I have admittedly only used the `TypeOperators` extension once, I
```

Comment:
As a matter of style, personal anecdotes like this are generally best avoided as they add length yet don't help the reader understand the proposal.